src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -553,3 +558,62 @@ export const getAttributes = (node: Element): Attributes => {
   const showValueOf = getAttributesDynamic(node)
   return { ...attributesStatic, showValueOf }
 }
+
+export const mapDefaultAggrMethod = (unit: string): string => {
```

Comment:
`transformDataAttributes` file is used for functions parsing DOM attributes for charts created with `data-netdata=".."` DOM attributes, perhaps it could be put just in some utils file?

Reply:
I've put it in `utils/fill-missing-data`